libs/SmartSync/SmartSync/Classes/Util/SFMruSyncDownTarget.m

```diff
@@ -41,6 +41,24 @@ @interface SFMruSyncDownTarget ()
 
 @implementation SFMruSyncDownTarget
 
+- (instancetype)initWithDict:(NSDictionary *)dict {
```

Comment:
Similar to `NSCoding`-style serialization/deserialization, we implement a serialization initializer, as well as a standard initializer (below).  We should consider moving to a standard `NSCoding` approach in the future, if it's not too bad to implement.